### DISCUSSION TITLE
Pretty name in alert when oom kill

### DISCUSSION THREAD

**Discussion by Franck31** [netdata/netdata#16116] [Q&A]
Hello Netdata Team,

I have a question regarding notifications for out-of-memory (OOM) events in our Kubernetes (K8s) setup. Currently, when we receive an OOM alert, it does not include the name of the cgroup that was terminated.

Example:

> "needs attention - oom kill
> mem.oom_kill (system)
> [oom kill = 1 kills]
> number of out of memory kills in the last 30 minutes"

Is there a way to include this information in the alert, or should I consider this as a feature request?

Thank you!
Cheers!

**Comment by ilyam8** [comment]
Hey, @Franck31. This alert should be silent - no notification sent. Or do you mean just an event in the Event tab?

**Reply by Franck31** [reply]
I have the alert notification enabled via Telegram or email to notify me if any pod crashes due to out-of-memory (OOM) issues. However, the current alert only provides limited information. To identify which pod caused the issue, I need to manually access the Kubernetes node and run 'dmesg' to check for relevant information.

Ideally, the alert should include details such as the cgroup or the specific process that triggered the OOM condition. This would greatly streamline the troubleshooting process and help me quickly identify and address the problem.

**Reply by ilyam8** [reply]
This specific alert has no sense in K8s env - it is the total number of oom kill events on the host, collected from `/proc/vmstat` (`oom_kill`). Netdata doesn't track container oom kills in K8s at the moment.
